### ISSUE TITLE
Contact class lacks enough specificity

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks-aldine#456]
The CSS for Aldine has a rule that targets the `.contact` class in order to style the contact form designed to go on the home page: https://github.com/pressbooks/pressbooks-aldine/blob/47ec1854b7f6a23529be0329ac7dc24e0eeb709c/assets/styles/components/_forms.scss#L109-L117

If a user creates a page with the title (& permalink) of 'Contact' the body element for the page itself also receives a `.contact` class, resulting in flex styling and padding being applied to the page. It can be overridden, but a better solution would be to add additional specification to the `.contact` class designed to target just the contact form and not other elements which may happen to have this class applied. I recommend using `#page`

**Comment by SteelWagstaff** [comment]
@cmurtagh and @tw77 -- this came up in our support checkin today. I've submitted a PR that I think should fix it by adding an additional selector to the rule which targets the `.contact` class: https://github.com/pressbooks/pressbooks-aldine/pull/457. The CSS workaround was simply to apply something like: 
```
body.contact {
	align-items: normal;
	padding: 0;
}
```

**Comment by cmurtagh** [comment]
PR Merged. @arzola I guess this means a new release of Aldine. Does this happen automagically, or do we need to trigger something manually?

**Comment by arzola** [comment]
Yes It cut a new release but it should be available for testing on dev right now

**Comment by SteelWagstaff** [comment]
I have a couple more PRs I'm hoping to open for Aldine in next week or so, if you want to hold off on cutting the new release?

**Comment by arzola** [comment]
sure @SteelWagstaff we can wait to release https://github.com/pressbooks/pressbooks-aldine/pull/472
